test/cypress/integration/tests/scenario-exec-spec.js

```diff
@@ -273,6 +369,137 @@ describe('Scenario Execution', function () {
     validateNetCharEvent(elemType, name, l, lv, pl, tp, tp-1, dist);
   }
 
+  // Save Replay Events
+  function testSaveReplayEvents(name, desc) {
+    cy.log('Save events');
+    click(meep.EXEC_BTN_EVENT);
+    click(meep.EXEC_BTN_EVENT_BTN_SAVE_REPLAY);
+    type(meep.MEEP_DLG_SAVE_REPLAY_NAME, name);
+    type(meep.MEEP_DLG_SAVE_REPLAY_DESCRIPTION, desc);
+    click(meep.MEEP_DLG_SAVE_REPLAY, 'Ok');
+    cy.wait(1000);
+    //no way to check except by executing auto-replay test that gets the file
+    click(meep.EXEC_BTN_EVENT_BTN_AUTO_REPLAY);
+    select(meep.EXEC_BTN_EVENT_BTN_AUTO_REPLAY_EVT_REPLAY_FILES, name);
+    cy.wait(500);
+    //check that by loading the saved replay event file that the description gets populated to confirm it was stored properly
+    //and check play button is enabled
+    cy.contains(desc)
+    verifyEnabled(meep.EXEC_BTN_EVENT_BTN_AUTO_REPLAY_BTN_REPLAY_START, true);
+  }
+
+  // Auto Replay
+  function testAutoReplay(name) {
+    cy.log('Auto Replay');
+
+    testAutoReplayExecution(name, false)
+    cy.wait(1000);
+    //confirm execution started
+    verifyEnabled(meep.EXEC_BTN_EVENT_BTN_AUTO_REPLAY_BTN_REPLAY_START, false);
+    cy.wait(75000);
+    //confirm it executed completely
+    verifyEnabled(meep.EXEC_BTN_EVENT_BTN_AUTO_REPLAY_BTN_REPLAY_START, true);
+
+    testAutoReplayExecution(name, true)
+    cy.wait(1000);
+    //confirm execution started
+    verifyEnabled(meep.EXEC_BTN_EVENT_BTN_AUTO_REPLAY_BTN_REPLAY_START, false);
+    cy.wait(75000);
```

Comment:
Does the actual replay time take close to 75 seconds?
Can this timer be reduced?

Reply:
used in conjonction with verifyEnabled... max should then be 75 sec.. could be quicker...

Reply:
verify enabled now used where the replay is executing once... otherwise, when in loop, still waiting for the whole 75s